pavelib/utils/test/utils.py

```diff
@@ -17,7 +17,7 @@ def clean_test_files():
     """
     Clean fixture files used by tests and .pyc files
     """
-    sh("git clean -fqdx test_root/logs test_root/data test_root/staticfiles test_root/uploads")
+    sh("git clean -fqdx test_root/logs test_root/data test_root/uploads")
```

Comment:
@andy-armstrong @clytwynec I'm confused. This PR does *not* break fasttest for bok-choy? Does it change the meaning/behavior of any features or options?

Also for the proposed PR for better support of fasttest, if we're not going to do this now, can we get a JIRA ticket in that outlines the work that needs to done there?